verification/curator-service/ui/cypress/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+    "extends": "../../../../common/tsconfig.json",
+    "compilerOptions": {
+        "strict": true,
+        "baseUrl": "../node_modules",
+        "target": "es5",
```

Comment:
this blog tends to indicate es6 is supported
https://glebbahmutov.com/blog/use-typescript-with-cypress/

can you remove this line and leave it to the common tsconfig to target es6?

you might have to change the "lib" below

Reply:
Done